devsquad-cockpit/laravel
# COC-485: update endpoint to api capture

## What?

- [COC-485](https://team-devsquad.atlassian.net/browse/COC-485)

## Why?

This PR updates the endpoint to use api/capture instead of the deprecated (now removed) webhook.

## Checklist:

#### Required to set up the environment to test this PR


#### Have you done all items below?


## How can it be tested?

- Try to send errors to Cockpit
- Check that it works as expected
